pkgs/applications/virtualization/hercules/default.nix

```
@@ -0,0 +1,18 @@
+{ stdenv, fetchurl, zlib, bzip2, pcre }:
```

These aren't being used 
```suggestion
{ stdenv, fetchurl }:
```